### PR TITLE
Added Ideas to showcase page

### PR DESCRIPTION
**Pull request by sainty0**

### CODE REVIEW

File: content/showcase/_index.html [hunk 1]
```diff
@@ -20,6 +20,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
                         link="https://github.com/nafisazizir/uquizzle"
                         class="showcase-element has-text-centered"
                     >}}
+                    {{< showcase-item
+                        src="/img/media/showcase/ideas.png"
+                        title="Ideas"
+                        caption="<p>Open-Source screenplay writing tool that let's you rearrange scenes and create storyboard images using AI.</p><p><b>2024 Hackathon Best Interdisciplinary</b></p>"
```
Comment (49Indium) [inline content/showcase/_index.html:26]: Typically, "Open-Source" is not a proper noun, so should be capitalised as "Open-source" when starting a sentence.

**Review by 49Indium — CHANGES_REQUESTED** [review]
Looks good. Just one small spelling change needed.

File: content/showcase/_index.html (hunk 1 above)
Comment (sainty0) [reply content/showcase/_index.html]: Fixed :)

**Review by 49Indium — APPROVED** [review]
Looks great! Will add to the website now.

**Pull request merged**